src/dmd/dtoh.d

```diff
@@ -0,0 +1,1256 @@
+// Compiler implementation of the D programming language
+// Copyright (c) 1999-2015 by Digital Mars
+// All Rights Reserved
+// written by Walter Bright
+// http://www.digitalmars.com
+// Distributed under the Boost Software License, Version 1.0.
+// http://www.boost.org/LICENSE_1_0.txt
+
+module dmd.dtoh;
+
+import core.stdc.stdio;
+import core.stdc.string;
+import core.stdc.ctype;
+
+import dmd.astcodegen;
+import dmd.arraytypes;
+import dmd.globals;
+import dmd.identifier;
+import dmd.visitor;
+
+import dmd.root.outbuffer;
+
+immutable string[] frontendSources = [
+    "access.d", "aggregate.d", "aliasthis.d", "apply.d", "argtypes.d", "arrayop.d", "arraytypes.d", "attrib.d",
+    "blockexit.d", "builtin.d",
+    "canthrow.d", "clone.d", "compiler.d", "complex.d", "cond.d", "console.d", "constfold.d", "cppmangle.d", "cppmanglewin.d", "ctfeexpr.d", "ctorflow.d",
+    "dcast.d", "dclass.d", "declaration.d", "delegatize.d", "denum.d", "dimport.d", "dinterpret.d",
+    "dmacro.d", "dmangle.d", "dmodule.d", "doc.d", "dscope.d", "dstruct.d", "dsymbol.d", "dsymbolsem.d", "dtemplate.d", "dversion.d",
+    "entity.d", "errors.d", "escape.d", "expression.d", "expressionsem.d",
+    "func.d",
+    "globals.d", "gluelayer.d",
+    "hdrgen.d",
+    "iasm.d", "id.d", "identifier.d", "impcnvtab.d", "imphint.d", "init.d", "initsem.d", "inline.d", "inlinecost.d", "intrange.d",
+    "json.d",
+    "lambdacomp.d", "lexer.d",
+    "mtype.d",
+    "nogc.d", "nspace.d",
+    "objc.d", "opover.d", "optimize.d",
+    "parse.d", "printast.d",
+    "safe.d", "sapply.d", "semantic2.d", "semantic3.d", "sideeffect.d", "statement.d",
+    "statementsem.d", "staticassert.d", "staticcond.d",
+    "target.d", "templateparamsem.d", "tokens.d", "traits.d", "typesem.d", "typinf.d",
+    "utf.d", "utils.d",
```

Comment:
Can't we use globbing here like we do for the dub package or `build.d`, s.t. we have less files to update whenever a file gets moved?

Reply:
Maybe, this part was carried over from the old PR.

For sure am open to design improvements - the most important part is the blacklisting of certain modules that we retain manual updates for.